swap_meet/vendor.py

```diff
@@ -1,2 +1,62 @@
+from swap_meet.item import Item
+
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=None):
+        if inventory is None:
+            self.inventory = []
+        else:
+            self.inventory = inventory
```

Comment:
another fun to do it `self.inventory = inventory if inventory is not None else []`